libmamba/src/api/install.cpp

```diff
@@ -567,8 +565,8 @@ namespace mamba
             fs::u8path pkgs_dirs(Context::instance().root_prefix / "pkgs");
             MultiPackageCache pkg_caches({ pkgs_dirs });
             prefix_data.add_packages(get_virtual_packages());
-            MRepo::create(
-                pool, prefix_data);  // Potentially re-alloc (moves in memory) Solvables in the pool
+            // Potentially re-alloc (moves in memory) Solvables in the pool
```

Comment:
I prefer that the comment be in the line it is commenting, mainly because future changes might insert lines here without seeing the comment only applies to that line.